JavaScript/JavaScriptSDK/Context/Session.ts

```diff
@@ -122,6 +112,32 @@ module Microsoft.ApplicationInsights.Context {
             }
         }
 
+        private initializeAutomaticSessionWithData(params) {
+            if (params.length > 0) {
+                this.automaticSession.id = params[0];
+            }
+
+            try {
+                if (params.length > 1) {
+                    var acq = +params[1];
+                    this.automaticSession.acquisitionDate = +new Date(acq);
+                    this.automaticSession.acquisitionDate = this.automaticSession.acquisitionDate > 0 ? this.automaticSession.acquisitionDate : 0;
+                }
+
+                if (params.length > 2) {
+                    var renewal = +params[2];
+                    this.automaticSession.renewalDate = +new Date(renewal);
+                    this.automaticSession.renewalDate = this.automaticSession.renewalDate > 0 ? this.automaticSession.renewalDate : 0;
+                }
+            } catch (e) {
+                _InternalLogging.throwInternalNonUserActionable(LoggingSeverity.WARNING, "Error parsing ai_session cookie, session will be reset: " + Util.dump(e));
```

Comment:
Is that the difference between `WARNING` and `CRITICAL`? 
I don't have a strong preference. We can change it back to `WARNING` later, if we decide that `CRITICAL` notifications are too noisy.